Gemfile

```diff
@@ -31,6 +31,9 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+# Use redis as cache
+gem 'redis', '~>3.2'
```

Comment:
Might be good to include redis as a prerequisite in the Getting Started section of the README